music_assistant/server/providers/fully_kiosk/__init__.py

```diff
@@ -146,22 +146,22 @@ async def loaded_in_mass(self) -> None:
     def _handle_player_update(self) -> None:
         """Update FullyKiosk player attributes."""
         player_id = self._fully.deviceInfo["deviceID"]
-        player = self.mass.players.get(player_id)
-        player.name = self._fully.deviceInfo["deviceName"]
-        # player.volume_level = snap_client.volume
-        for volume_dict in self._fully.deviceInfo.get("audioVolumes", []):
-            if str(AUDIOMANAGER_STREAM_MUSIC) in volume_dict:
-                volume = volume_dict[str(AUDIOMANAGER_STREAM_MUSIC)]
-                player.volume_level = volume
-                break
-        current_url = self._fully.deviceInfo.get("soundUrlPlaying")
-        player.current_item_id = current_url
-        if not current_url:
-            player.state = PlayerState.IDLE
-        player.available = True
-        self.mass.players.update(player_id)
-
-    async def get_player_config_entries(self, player_id: str) -> tuple[ConfigEntry]:
+        if player := self.mass.players.get(player_id):
```

Comment:
you can also turn this into a guard clause instead:

```suggestion
        if(player := self.mass.players.get(player_id)) is None:
            return
```